tests/test_plotting.py

```diff
@@ -486,22 +486,23 @@ def test_plot_scatter(close_figures) -> None:
         ),
         Axes,
     )
-    check(
-        assert_type(
-            IRIS_DF.plot.scatter(x="SepalLength", y="SepalWidth", subplots=True),
-            npt.NDArray[np.object_],
-        ),
-        np.ndarray,
-    )
-    check(
-        assert_type(
-            IRIS_DF.plot(
-                x="SepalLength", y="SepalWidth", subplots=True, kind="scatter"
+    with pytest.warns(UserWarning):
```

Comment:
I don't see why this is necessary.

Reply:
See the failing CI tests https://github.com/pandas-dev/pandas-stubs/actions/runs/9244454746/job/25429762574. A new version of matplotlib is causing new test failures with the warning.